cmd/renterd/main.go

```diff
@@ -40,9 +52,29 @@ on how to configure and use renterd.
 func main() {
 	log.SetFlags(0)
 
-	// load the config
-	cfg, network, genesis, err := loadConfig()
-	checkFatalError("failed to load config", err)
+	// load config file
+	configPath := tryLoadConfig()
```

Comment:
So there is one more thing I noticed when testing and I think it's the behavior we want but I'm bringing it up just to highlight it.
When you have a `renterd.yml` file at `~/Application Support/renterd/renterd.yml` that sets the data directory to `~/renterd`, running `renterd config` will also default to `~/renterd` and ask you whether you want to change it.
If you don't have that config file, it will propose `~/Application Support/renterd` as the data dir.

So the question here is whether we want to config file of which we just said we are willing to override it, influence the default path of the data dir we are about to pick.

Reply:
I would say so because it's consistent with how we treat the seed phrase.